config/jobs/kubernetes-sigs/sig-windows/release-1.22-windows.yaml

```diff
@@ -56,17 +56,18 @@ periodics:
         privileged: true
 - name: ci-kubernetes-e2e-capz-azure-windows-dockershim-1-22
   interval: 3h
```

Comment:
Do we want to run these every 3 hours?

Reply:
no, updated it to 12 hours for the release